Fixed #35350 -- Fixed save() with pk set on models with GeneratedFields. (#18058)
django/django: docs/releases/5.0.5.txt

Bugfixes
========

> * Fixed a bug in Django 5.0 that caused a crash of ``Model.save()`` for
>   models with a ``GeneratedField`` when creating a model instance while
>   providing a primary key (:ticket:`35350`).

Reviewer: Just an optional suggestion!

```suggestion
* Fixed a bug in Django 5.0 that caused a crash of ``Model.save()`` when
  creating an instance of a model with a ``GeneratedField`` and providing a
  primary key (:ticket:`35350`).
```
